src/components/Flashlight.js

```diff
@@ -7,7 +7,8 @@ import './style/Flashlight.css';
 
 class Flashlight extends React.Component {
 
-    showFlashlight = () => {
+    showFlashlight = e => {
+        e.preventDefault();
```

Comment:
I can figure this out with you later (the mobile compat), for now let's table it (but maybe add a `// TODO` comment)